apps/codelab/src/app/codelabs/angular/intro/intro.component.css

```diff
@@ -0,0 +1,4 @@
+.image {
+  width: 100%;
+  height: calc(100% - 10px);
```

Comment:
Is this really needed? What prevents if from being 100%?

Reply:
The purpose of `calc` was to not fit on the menu on the right